canonical-web-and-design/maas-ui
# Update commissioning field options on change

## Done
Updated the settings commissioning form so that the second fields options update when the first option changes. Fixes [#1414](https://github.com/canonical-web-and-design/MAAS-squad/issues/1414)

## QA
- Go to `/settings/configuration/commissioning`
- See that the field labelled `Default minimum kernel version` options are related to the above field (i.e. same version number 18.04 etc.)
- Change the field labelled `Default Ubuntu release used for commissioning` and see that the options in the field beneath are updated to reflect the new value
